src/components/searchDashboard/Search.jsx

```diff
@@ -6,9 +6,9 @@ import { Box } from '@mui/system';
 import SearchIcon from '@mui/icons-material/Search';
```

Comment:
in line 5, import Box from material not system,

Reply:
done